system/MagicMemory.cpp

```diff
@@ -67,7 +67,7 @@ namespace atlas
                 }
                 break;
             case SupportedDevices::BLOCK_CHAR:
-                sparta_assert(false, "TODO: Block char is unsupported");
+                //sparta_assert(false, "TODO: Block char is unsupported");
```

Comment:
This assert is firing with the -v- tests; I commented it out for now to see how far the tests go.  They still all fail. :laughing: